README.md

```diff
@@ -81,9 +78,9 @@ String fingerprint = fingerprintAndroidAgent.getFingerprint(
 String deviceId = fingerprintAndroidAgent.deviceId();
```

Comment:
again, unclear what is deviceId and what is fingerprint here. Why is it called "agent"? Is it a service that runs alongside the main app?